### ISSUE TITLE
Allow custom messages on footer

### ISSUE THREAD

**Issue by TheoLechemia** [swimlane/ngx-datatable#1320]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[ x] feature request
[ ] support request => Please do not submit support request here, post on Stackoverflow or Gitter
```

**Current behavior**
It's actually possible to put custom messages on the datatable footer, but there is alway the total number displaying.

When I put on "12 row(s) on 19 total"  on ``totalMessage`` I got the following result because the ``rowCount`` is hard coded in the footer component

![image3379](https://user-images.githubusercontent.com/15265745/36985803-84faeb7e-2098-11e8-9f4b-83acb05ca190.png)

Cf FooterComponent:
``
      <div class="page-count" *ngIf="!footerTemplate">
        <span *ngIf="selectedMessage">
          {{selectedCount.toLocaleString()}} {{selectedMessage}} / 
        </span>
        {{rowCount.toLocaleString()}} {{totalMessage}}
      </div>
``

**Expected behavior**

Make possible to put custom messages without the "total" number

* **Table version:** 0.8.x
10.4.0
* **Angular version:** 2.0.x
4.0

**Comment by TheoLechemia** [comment]
Anyone on this issue ?

**Comment by mmdsharifi** [comment]
Yes @TheoLechemia , I want to localize (translate) words like "total" in footer what is your solution ?
